eng/scripts/Prepare-Release.ps1

```diff
@@ -1,7 +1,8 @@
 [CmdletBinding()]
 param(
     [Parameter(Mandatory=$true)]
-    $package
+    [string]$package,
```

Comment:
Is this mandatory? If so, use the same `[Parameter]` attribute as above. In fact, generally you should use these for full powershell support.

Reply:
Nope, it's not.